styles.css

```diff
@@ -0,0 +1,547 @@
+  letter-spacing: 0.642px;
+}
+
+.items h3 {
+  width: 40%;
+  height: auto;
+  margin: 5px;
+  color: #ff5438;
+  font-family: "Lato", sans-serif;
+  font-size: 14.271px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24.975px;
+  letter-spacing: -0.143px;
+}
+
+.items img {
+  margin: 5px;
+  width: 57.085px;
+  height: 51.733px;
+}
+
+.section2 h2 {
+  margin: 20px 0 20px 0;
+  color: #fff;
+  font-family: "Lato", sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px; /* 140% */
+  letter-spacing: -0.8px;
+}
+
+.join {
+  display: flex;
+  align-items: center;
+  height: 70px;
+  background-color: #ff5438;
+  width: 80%;
+  padding: 20px 20px;
+  margin: 5px 0 20px 0;
+  justify-content: center;
+  gap: 10px;
+}
+
+.section3 {
+  padding-top: 30px;
+}
+
+.speakers {
+  display: flex;
+  justify-content: center;
+  height: 80px;
+  color: #000;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: -1px;
+}
+
+.speakers1,
+.speakers2,
+.speakers3 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.speakeritems {
+  width: 100%;
+  display: flex;
+  margin: 10px;
+}
+
+.containerB {
+  padding: 10px;
+}
+
+.speakeritems img {
+  width: 107px;
+  height: 106.399px;
+}
+
+.itemsA {
+  padding: 5px 2px 20px 10px;
+}
+
+.itemsA h5 {
+  color: #000;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  letter-spacing: -0.3px;
+}
+
+.itemsA span {
+  color: #e32116;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 13.943px;
+  letter-spacing: -0.1px;
+  margin-bottom: 20px;
+}
+
+.itemsA p {
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 17.078px; /* 189.753% */
+  letter-spacing: -0.45px;
+  width: 229px;
+}
+
+.container5 img {
+  width: 200px;
+  height: 50.47px;
+}
+
+.section4 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  gap: 20px;
+  background: #414246;
+  justify-content: center;
+  height: 400px;
+}
+
+.container5 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.section4 h6 {
+  color: #8f8f91;
+  font-family: "Lato", sans-serif;
+  font-size: 30px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: -0.3px;
+  width: 98px;
+  height: 34px;
+}
+
+@media screen and (min-width: 768px) {
+  .header .header-list1 {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    background: #3e3e3e;
+    color: #fff;
+    gap: 10px;
+    padding: 1px 190px 1px 0;
+    font-size: 15px;
+    height: 30px;
+    font-weight: 400;
+  }
+
+  .containerA {
+    padding: 5px;
+    margin-left: 180px;
+    width: 70%;
+    height: 100%;
+    flex-shrink: 0;
+  }
+
+  .section1 {
+    display: flex;
+    height: 400px;
+    padding: 50px 0 0 35px;
+  }
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement according to which you can make little changes or adjustments of your own choice but can't change it completely but as I can see that your design looks different than actual, which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://github.com/ulises2607/cayafate-project/assets/66476329/cd029716-65ab-45b5-8e7d-6886f05d48af)


### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-22 16-21-56](https://github.com/ulises2607/cayafate-project/assets/66476329/e71bdc71-6a09-48bd-a21a-2f6f722cbb08)

</td>
<td>

![Screenshot from 2023-08-03 01-33-22](https://github.com/enocol/capstone/assets/66476329/e6c2d3df-49b5-4e18-9e5f-86d4a99d176a)

</td>
</tr>
</table>

### Mobile Navigation Page
- [ ] Please add the close button in your mobile navigation, so that if the the user wants to stay on the same page, he should be able to close the navigation menu and explore things. But it seems like there is no button to close the overlay, once it is opened.

![image](https://github.com/enocol/capstone/assets/66476329/fccdf8d1-bece-4abc-9049-db4ab9093b7f)

Reply:
@skyv26  Please could you provide the link to the Figma design you are referring to because as far I am aware, the is no Figma design link provided to copy from.

Reply:
I will check why you did not see a close button. There was a working closed button but I will check why you did not see this 
  and fix it if you confirm where you were examining the design.

Reply:
@skyv26 Please you will have to call or give me some more information because, since you have been referring to a Figma design, the summit title is an image with no CSS properties to copy. I am not able to implement your recommendation with the little information you have given me.

Reply:
Hi @enocol, Sorry for using `Figma` keyword instead of `behance` design. But the rest of the things  that I mentioned are still the same. You have to follow the design given by Microverse. That design creator is Cindy Shin. Kindly implement the changes that I gave regardless of the wrong keyword `Figma`.

Reply:
I checked your design on 375px screen size which is the minimum screen size for the majority of the mobile.